projects/fsastorefrontlib/src/cms-components/assets-table/assets-table.component.html

```diff
@@ -10,49 +15,39 @@
         *ngFor="let asset of assets"
         (click)="resolveAssetUrl(asset)"
       >
-        <td class="d-flex d-md-table-cell">
-          <div class="mobile-label text-uppercase pr-2 d-md-none">
-            {{ headings[0] }}:
-          </div>
-          <span class="notice text-break">
-            {{ asset?.quoteId }}
-            {{ asset?.contractNumber }}
-            {{ asset?.claimNumber }}
-          </span>
-        </td>
-        <td class="d-flex d-md-table-cell">
-          <div class="mobile-label text-uppercase pr-2 d-md-none">
-            {{ headings[1] }}:
-          </div>
-          <span class="text-break">
-            {{ asset?.defaultCategory?.name }}
-            {{ asset?.categoryData?.name }}
-            {{ asset?.insurancePolicy?.categoryData?.name }}</span
-          >
-        </td>
-        <td class="d-flex d-md-table-cell">
+        <td
+          class="d-flex d-md-table-cell"
+          *ngFor="
+            let value of dataByAssetType[selectedAsset].values;
+            let i = index
+          "
+        >
           <div class="mobile-label text-uppercase pr-2 d-md-none">
-            {{ headings[2] }}:
+            {{ dataByAssetType[selectedAsset].headings[i] | cxTranslate }}:
           </div>
-          <span class="text-break text-capitalize">
-            {{ asset?.paymentFrequency }}
-            {{ asset?.insurancePolicy?.paymentFrequency }}</span
-          >
-        </td>
-        <td class="d-flex d-md-table-cell">
-          <div class="mobile-label text-uppercase pr-2 d-md-none">
-            {{ headings[3] }}:
-          </div>
-          <span class="text-break text-capitalize">
-            {{ asset?.quoteStatus }}
-            {{ asset?.policyStatus }}
-            {{ asset?.claimStatus }}</span
+          <span
+            class="notice text-break"
+            (click)="value.startClaim ? startClaim($event, asset) : false"
```

Comment:
This should all be resolved in `startClaim` method in component ts file. We should avoid resolving logic in the template, whenever possible.